app/models/pipeline_run.rb

```diff
@@ -814,9 +883,15 @@ def check_and_enqueue(output_state)
     if output_ready?(output)
       output_state.update(state: STATUS_LOADING_QUEUED)
       Resque.enqueue(ResultMonitorLoader, id, output)
+    # check if job is done more than a minute ago
     elsif finalized? && pipeline_run_stages.order(:step_number).last.updated_at < 1.minute.ago
-      # check if job is done more than a minute ago
-      output_state.update(state: STATUS_FAILED)
+      # HACK: don't fail for missing insert, size metrics, they are optional
+      #   Doing this may result in missing some errors
+      if output_state.output == "insert_size_metrics"
+        output_state.update(state: STATUS_LOADED)
```

Comment:
I didn't get the idea here. I don't like ignoring errors either. I would like to understand better why would we need to do that. I agree with Charles, we should try finding a different mechanism.